packages/compliance-tests/src/peer-id/index.js

```diff
@@ -382,29 +378,24 @@ module.exports = (common) => {
 
       it('missmatch private - public key', async () => {
         const digest = await k1.public.hash()
-        expect(() => {
-          factory.createFromJSON({
-            id: digest,
-            pubKey: k1,
-            privKey: k2.public
-          }) // eslint-disable-line no-new
-        }).to.throw(/inconsistent arguments/)
+        expect(factory.createFromJSON({
```

Comment:
You need to await on or return the promise returned when using `eventually`:

```suggestion
        await expect(factory.createFromJSON({
```